JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreen.kt

```diff
@@ -444,24 +504,50 @@ private fun PostListDivider() {
     )
 }
 
-@Preview("Home screen")
-@Preview("Home screen (dark)", uiMode = UI_MODE_NIGHT_YES)
-@Preview("Home screen (big font)", fontScale = 1.5f)
-@Preview("Home screen (large screen)", device = Devices.PIXEL_C)
+@Preview("Home screen drawer")
+@Preview("Home screen drawer (dark)", uiMode = UI_MODE_NIGHT_YES)
+@Preview("Home screen drawer (big font)", fontScale = 1.5f)
 @Composable
-fun PreviewHomeScreen() {
+fun PreviewHomeScreenWithDrawer() {
     val posts = runBlocking {
         (BlockingFakePostsRepository().getPosts() as Result.Success).data
     }
     JetnewsTheme {
         HomeScreen(
             uiState = HomeUiState(posts = posts),
+            showNavRail = false,
             onToggleFavorite = { /*TODO*/ },
             onRefreshPosts = { /*TODO*/ },
             onErrorDismiss = { /*TODO*/ },
             navigateToArticle = { /*TODO*/ },
+            navigateToInterests = { /*TODO*/ },
             openDrawer = { /*TODO*/ },
             scaffoldState = rememberScaffoldState()
         )
     }
 }
+
+@Preview("Home screen navrail", device = Devices.PIXEL_C)
+@Preview("Home screen navrail (dark)", device = Devices.PIXEL_C, uiMode = UI_MODE_NIGHT_YES)
+@Preview("Home screen navrail (big font)", device = Devices.PIXEL_C, fontScale = 1.5f)
+@Composable
+fun PreviewHomeScreenWithNavRail() {
+    val posts = runBlocking {
+        (BlockingFakePostsRepository().getPosts() as Result.Success).data
+    }
+    JetnewsTheme {
+        Surface {
+            HomeScreen(
+                uiState = HomeUiState(posts = posts),
+                showNavRail = true,
+                onToggleFavorite = { /*TODO*/ },
```

Comment:
nit: This should be NO-OP instead of TODO (it's a keyword AS uses for actual to-dos)

Reply:
👍  changed all of them to `NO-OP`